dump/src/util/dump/GroupIndex.java

```diff
@@ -329,7 +329,18 @@ protected void initLookupMap() {
       }
    }
 
-@Override
+   @Override
+   protected void clearLookupMap() {
+      if ( _fieldIsInt ) {
+         _lookupInt.clear();
+      } else if ( _fieldIsLong ) {
+         _lookupLong.clear();
+      } else {
+         _lookupObject.clear();
+      }
```

Comment:
Shouldn't we simply call `initLookupMap` here?

Reply:
We could, but an appropriately sized container is already there, ready to receive all the dump's contents without incrementally rehashing all the time...

Reply:
Well, there's a better solution for that, so let's pass on the additional complexity there.